### ISSUE TITLE
deno install/run parity

### ISSUE THREAD

**Issue by fsenart** [denoland/deno#6921]
**Deno version:** 

```
deno 1.2.1
v8 8.5.216
typescript 3.9.2
```

**Search terms:** install no-check

**code:**

```
deno install --no-check https://deno.land/std/examples/welcome.ts
```

**Expected behavior:**

```
Download https://deno.land/std/examples/welcome.ts
Warning Implicitly using master branch https://deno.land/std/examples/welcome.ts
Check https://deno.land/std/examples/welcome.ts
✅ Successfully installed welcome
/foo/.deno/bin/welcome
```

**Actual behavior:**

```
error: Found argument '--no-check' which wasn't expected, or isn't valid in this context

USAGE:
    deno install [OPTIONS] <cmd>...

For more information try --help
```

**Discussion:**

As I understand it, `deno install` serves as a *shortcut*/*wrapper* around a more verbose `deno run` command. I expect `install` to accept all the flags accepted by `run`.

**Comment by nayeemrmn** [comment]
In this case, `--no-check` should not only be forwarded to the written `deno run` command but also affect the pre-caching that is done as part of `deno install`.

**Comment by JayHelton** [comment]
@nayeemrmn im interested in grabbing this and giving it a shot. Have you already started on this by chance?

**Comment by nayeemrmn** [comment]
@JayHelton No, go ahead! This needs to be part of the minor release on the 13th, I'm guessing there'll be one more patch release in between before this change can land.

**Comment by JayHelton** [comment]
@nayeemrmn can you elaborate a bit more on the second half of your comment? 
> but also affect the pre-caching that is done as part of deno install

**Comment by JayHelton** [comment]
@nayeemrmn gentle reminder on the above! I have a change for passing through the no check arg to the script created, but havent quite caught onto what you meant by affecting the pre caching. I have debugged through most of deno and wasnt quite clear! Thanks again!

**Comment by bartlomieju** [comment]
Closed in #6948
